doc/cdi-operator-olm.md

```diff
@@ -35,46 +35,64 @@ DOCKER_REPO=<repo> DOCKER_TAG=<docker tag> PULL_POLICY=<pull policy> VERBOSITY=<
 QUAY_NAMESPACE=<quay namespace> QUAY_REPOSITORY=<quay repo> QUAY_USERNAME=<quay username> QUAY_PASSWORD=<quay password> CSV_VERSION=<csv version > make olm-push
 ```
 #### Install OLM and Marketplace operators on cluster
-- Install OLM operator from cloned operator-lifecycle-manager repo 
+- Install OLM operator from cloned operator-lifecycle-manager repo and wait untill all pods are Running and Ready. 
 
 ```bash
 kubectl apply -f $GOPATH/src/github.com/operator-framework/operator-lifecycle-manager/deploy/upstream/quickstart/olm.yaml
 ```
-- Install marketplace operator from cloned operator-marketplace repo
+- Install marketplace operator from cloned operator-marketplace repo and wait untill all pods are Running and Ready.
 ```bash
 kubectl apply -f $GOPATH/src/github.com/operator-framework/operator-marketplace/deploy/upstream/ --validate=false
 ```
-###CDI installation by means of OLM and Marketplace operators
+- Wait till cdi marketplace is Running and Ready.
+```bash
+kubectl get pods -n marketplace 
+NAME                                   READY   STATUS    RESTARTS   AGE
+cdi-7c7fc4f774-bdbsh                   1/1     Running   0          37s
+marketplace-operator-d8cc985d4-mv7xp   1/1     Running   0          2m40s
+
+```
+### CDI installation by means of OLM and Marketplace operators
 - Install CDI operatorsource manifest that specifies the location of CDI OLM bundle in quay
 ```bash
-kubectl apply -f _out/manifests/release/olm/cdi-operatorsource.yaml_
+kubectl apply -f _out/manifests/release/olm/cdi-operatorsource.yaml
 ```
 - Handle marketplace namespace workarouond
 
   Move _catalogsourceconfig.operators.coreos.com/cdi_ from _markeplace_ namespace to _olm_ namespace by modifying *targetNamespace* field to 'olm' from 'marketplace'
 ```bash
-cluster/kubectl.sh get operatorsource,catalogsourceconfig,catalogsource,subscription,installplan --all-namespaces
-cluster/kubectl.sh edit catalogsourceconfig.operators.coreos.com/cdi -n marketplace
+kubectl get operatorsource,catalogsourceconfig,catalogsource,subscription,installplan --all-namespaces
+kubectl edit catalogsourceconfig.operators.coreos.com/cdi -n marketplace
 ```
 - Create CDI namespace
 ```bash
 kubectl create ns cdi 
 ```
 - Configure namespace to be allowed to create operators there
 ```bash
-cluster/kubectl.sh apply -f _out/manifests/release/olm/operatorgroup.yaml
+kubectl apply -f _out/manifests/release/olm/operatorgroup.yaml
 ```
 - Install subscription that will point from which channel the app is downloaded
 ```bash
-cluster/kubectl.sh apply -f  _out/manifests/release/olm/cdi-subscription.yaml
+kubectl apply -f  _out/manifests/release/olm/cdi-subscription.yaml
+```
+- Verify cdi installation plan was created
```

Comment:
Verify CDI installation plan was created
Try to keep CDI in caps as it is an abbreviation of Containerized Data Importer. I usually try to fully spell out the first CDI fully with (CDI) behind it, to give some context what CDI means.

Reply:
Done